scalacheck/src/main/scala/pathy/scalacheck/PathyArbitrary.scala

```diff
@@ -40,6 +40,9 @@ object PathyArbitrary {
   implicit val arbitraryDirName: Arbitrary[DirName] =
     Arbitrary(genDirName[RandomSeg])
 
+  implicit val arbitraryPathCodec: Arbitrary[PathCodec] =
+    Arbitrary(Gen.oneOf(posixCodec,windowsCodec))
```

Comment:
Seems like this is unused now. I don't think we need it anymore, now that you have validateIsLossless.